.github/workflows/pr_ci.yml

```diff
@@ -27,8 +27,6 @@ jobs:
     - name: Build
       run: |
         dotnet build -c Release -f net8.0
```

Comment:
I am wondering if not setting `-f` would have just built all tfms.

Reply:
Yes but when multiple tfms are built concurrently there are some issues when it outputs files.